PocoDoc/Makefile

```diff
@@ -10,6 +10,10 @@ objects = DocWriter PocoDoc
 
 target         = PocoDoc
 target_version = 1
-target_libs    = PocoCppParser PocoFoundation PocoXML PocoUtil
+target_libs    = PocoCppParser PocoDataSQLite PocoData PocoFoundation PocoXML PocoUtil
```

Comment:
Newly added data libraries shall be added to CMake also. PocoDoc doers not compile with CMake currently.

Reply:
Fixed in ef13d0fab675b72f6db2bc185d1ecab1a2494b31